2-mandatory/1-weather-report.js

```diff
@@ -12,7 +12,10 @@
 */
 
 function getTemperatureReport(cities) {
-    // TODO
+    let temparatureStatements  = [];
+    for (let i = 0; i < cities.length; i++){
```

Comment:
When you get the solutions for this week, take a look at the example solution. Yours works and is a correct answer; the mentor chose a different strategy. Can you think what the advantages of the mentor's choices are?